src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -78,31 +92,95 @@ export default function FundingForm({ onFinish }: { onFinish: VoidFunction }) {
 
   const [targetCurrency, setTargetCurrency] =
     useState<V2CurrencyOption>(V2_CURRENCY_ETH)
-  const { fundAccessConstraints, fundingCycleData, payoutGroupedSplits } =
-    useAppSelector(state => state.editingV2Project)
-
-  const [durationEnabled, setDurationEnabled] = useState<boolean>(
-    parseInt(fundingCycleData?.duration ?? '0') > 0,
-  )
+  const [durationEnabled, setDurationEnabled] = useState<boolean>(false)
   const [targetType, setTargetType] = useState<TargetType>('specific')
 
+  // Load redux state (may not exist)
+  const { fundAccessConstraints, fundingCycleData, payoutGroupedSplits } =
+    useAppSelector(state => state.editingV2Project)
   const fundAccessConstraint =
     getDefaultFundAccessConstraint<SerializedV2FundAccessConstraint>(
       fundAccessConstraints,
     )
 
+  // Load queued FC data of project
+  const { data: queuedFundingCycleResponse } = useProjectQueuedFundingCycle({
+    projectId,
+  })
+
+  const [queuedFundingCycle] = queuedFundingCycleResponse || []
+
+  let effectiveFundingCycle: V2FundingCycle | undefined
+  // If duration is 0, use current FC (not queued)
+  if (!fundingCycle?.duration || fundingCycle?.duration.eq(0)) {
+    effectiveFundingCycle = fundingCycle
+  } else {
+    effectiveFundingCycle = queuedFundingCycle
+  }
+
+  const { data: queuedDistributionLimitData } = useProjectDistributionLimit({
+    projectId,
+    configuration: effectiveFundingCycle?.configuration.toString(),
+    terminal: primaryTerminal,
+  })
+  const [queuedDistributionLimit, queuedDistributionLimitCurrency] =
+    queuedDistributionLimitData ?? []
+
+  const { data: queuedPayoutSplits } = useProjectSplits({
+    projectId,
+    splitGroup: ETH_PAYOUT_SPLIT_GROUP,
+    domain: effectiveFundingCycle?.configuration?.toString(),
+  })
+
   const ETHPaymentTerminalFee = useETHPaymentTerminalFee()
   const feeFormatted = ETHPaymentTerminalFee
     ? formatFee(ETHPaymentTerminalFee)
     : undefined
 
+  // Load queued FC data (if it exists) into redux
+  useEffect(() => {
```

Comment:
@johnnyd-eth I was actually thinking that all this default-redux-setting stuff would live in either:
- src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx
- src/components/v2/V2Project/V2ProjectReconfigureModal/V2ReconfigureModalTrigger.tsx

the reason being that queuedFC stuff isn't relevant to the create flow, so we don't need all this extra code in this FundingForm component.

WDYT

Reply:
Yeah true ok agreed